codec-dns/src/main/java/io/netty/handler/codec/dns/DefaultDnsRecordDecoder.java

```diff
@@ -106,7 +106,19 @@ protected DnsRecord decodeRecord(
      * @param in the byte buffer containing the DNS packet
      * @return the domain name for an entry
      */
-    protected String decodeName(ByteBuf in) {
+    protected String decodeName0(ByteBuf in) {
```

Comment:
Yes

> Am 16.06.2016 um 22:38 schrieb Xiaoyan Lin <notifications@github.com>:
> 
> In codec-dns/src/main/java/io/netty/handler/codec/dns/DefaultDnsRecordDecoder.java:
> 
> > @@ -106,7 +106,19 @@ protected DnsRecord decodeRecord(
> >       * @param in the byte buffer containing the DNS packet
> >       * @return the domain name for an entry
> >       */
> > -    protected String decodeName(ByteBuf in) {
> > +    protected String decodeName0(ByteBuf in) {
> Why need to create this decodeName0 method? Is it for providing a hook method to allow people to change the behavior?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>